workflows/crisprseq_targeted.nf

```diff
@@ -578,16 +578,6 @@ workflow CRISPRSEQ_TARGETED {
         true
     )
     .bam
-    .map {
```

Comment:
I added this because I didn't want to use the `meta.id`, which will be the name of the sample, but the name of the file which will be the name of the reference (or any other name the user gave to this file). When adding `template-align` to the name, if I resume the pipeline, it keeps adding `template-align`, that's why I wrote this workaround with map.

Was this bit causing the error? 

I think your fix will work good (I haven't tested it), and I agree using `meta.id` is easier and cleaner, and users don't need to inspect this file so it is not a big deal if we rename them. 
TL;DR: I agree with the change 😄

Reply:
No I think your fix will work fine, thanks :)